SchedulerJobs/SchedulerJobs.Services/JobHistoryService.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BookingsApi.Client;
+
+namespace SchedulerJobs.Services
+{
+    public interface IJobHistoryService
```

Comment:
not sure if this project is structured like this, but optionally we could move this into its separate file, but wouldnt block the pr for that change